src/Ocis.php

```diff
@@ -217,7 +216,7 @@ public static function isConnectionConfigValid(array $connectionConfig): bool
      *         The function is public to make it testable.
      * @phpstan-param ConnectionConfig $connectionConfig
      */
-    public static function createGuzzleConfig(array $connectionConfig, string $accessToken): array
+    public static function createGuzzleConfig(array $connectionConfig, ?string $accessToken): array
```

Comment:
still, `accessToken` should never be null at this stage
```suggestion
    public static function createGuzzleConfig(array $connectionConfig, string $accessToken): array
```